app/lib/three_scale/money.rb

```diff
@@ -15,6 +17,10 @@ class Money
     attr_reader :currency
     attr_reader :date
 
+    def self.cents(amount, currency, date = nil)
+      new(amount.to_d / 100, currency, date)
```

Comment:
Yes BigDecimal :+1: 

By the way our ThreeScale::Money does not handle this well (new issue to track)

```ruby
m1 = ThreeScale::Money.new(1.4, 'EUR')
m2 = ThreeScale::Money.new(1.2, 'EUR')

(m1 + m2).to_f # => => 2.5999999999999996
```
Lets fix it later